frontend/src/components/kl-add-item.js

```diff
@@ -28,7 +28,18 @@ export default {
             }).catch(err => {
                 if (err.status === 400 && err.body.code === 1) {
                     let renderedErrors = renderValidationErrors(err.body.errors);
-                    renderedErrors.forEach(error => toastr.error(this.$t('errors.' + error.id, error.props)));
+                    renderedErrors.forEach(error => {
+                        if (error.id === 'mustBeLessThan')
+                            toastr.error(this.$t('errors.' + error.id, error.props));
+                        else {
+                            let res;
+                            if (error.count === 1)
+                                res = 1;
```

Comment:
I use  `$t` for `mustBeLessThan` because I need template for separated properties. 
And `$tc` for other errors where needed single/plural
![default](https://user-images.githubusercontent.com/26686649/29495704-ab2cbe98-85ee-11e7-9417-ca27ed4efd66.PNG)